nilearn/interfaces/tests/test_bids.py

```diff
@@ -588,6 +590,8 @@ def test_save_glm_to_bids_contrast_definitions(
         "contrast-aaaMinusBbb_stat-t_statmap.nii.gz",
         "contrast-aaaMinusBbb_stat-variance_statmap.nii.gz",
         "contrast-aaaMinusBbb_stat-z_statmap.nii.gz",
+        "contrast-aaaMinusBbb_results.tsv",
```

Comment:
Should the threshold info be in the filename? Not exactly sure how best to do that, but I can imagine people sharing results with different thresholds.

Reply:
Dumping info in the sidecar json for now.

Reply:
For now I went for. Do not put that info in the filename and always overwrite previous results.
To keep things simple.
If users want more flexibility, let's wait for them to ask for it rather than read the mind of hypothetical future users.